search/src/test/scala/weco/api/search/services/WorksServiceTest.scala

```diff
@@ -82,52 +58,29 @@ class WorksServiceTest
 
     it("returns an empty result set when asked for a page that does not exist") {
       assertListOrSearchResultIsCorrect(
-        allWorks = indexedWorks(count = 3),
+        allWorks = visibleWorks,
         expectedWorks = Seq(),
-        expectedTotalResults = 3,
+        expectedTotalResults = visibleWorks.length,
         worksSearchOptions = createWorksSearchOptionsWith(pageNumber = 4)
       )
     }
 
-    it("does not list invisible works") {
```

Comment:
I removed this test because it's now covered by "gets records in Elasticsearch" – that earlier test indexes a mix of visible/invisible/redirected/deleted works, and checks that only the visible ones come back.